packages/i18n/README.md

```diff
@@ -61,79 +55,19 @@ _Returns_
 
 <a name="_n" href="#_n">#</a> **\_n**
 
-Translates and retrieves the singular or plural form based on the supplied
-number.
-
-_Related_
-
--   <https://developer.wordpress.org/reference/functions/_n/>
-
-_Parameters_
-
--   _single_ `string`: The text to be used if the number is singular.
--   _plural_ `string`: The text to be used if the number is plural.
--   _number_ `number`: The number to compare against to use either the singular or plural form.
--   _domain_ `[string]`: Domain to retrieve the translated text.
-
-_Returns_
-
--   `string`: The translated singular or plural form.
+Undocumented declaration.
```

Comment:
It's unfortunate that the doc parser can't find these definitions. If we decide to move ahead with this we should be able to find a way around this.